task 06/task6-2/ConsoleApp5/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace ConsoleApp4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+
+            var n = Program.vvedenie("x");
+            var m = Program.vvedenie("y");
+            Console.WriteLine(Program.proverka(n, m));
+        }
+
+        public static bool proverka(double a, double b)
+        {
+            return (a >= 2 && b >= 0 || a >= 1 && b <= -1);
+        }
+        public static double vvedenie(string z)
```

Comment:
Стиль! Приучите себя называть методы с большой буквы
```suggestion
        public static double Vvedenie(string z)
```